go.mod

```diff
@@ -1,22 +1,25 @@
 module github.com/utilitywarehouse/wiresteward
 
-go 1.15
+go 1.16
 
 require (
 	github.com/coreos/go-iptables v0.5.0
 	github.com/google/go-cmp v0.5.4
-	github.com/mdlayher/netlink v1.3.0 // indirect
+	github.com/mdlayher/netlink v1.3.1 // indirect
 	github.com/mdlayher/promtest v0.0.0-20200528141414-3c8577d47d5c
+	github.com/mikioh/ipaddr v0.0.0-20190404000644-d465c8ab6721 // indirect
 	github.com/prometheus/client_golang v1.9.0
+	github.com/prometheus/common v0.17.0 // indirect
 	github.com/prometheus/procfs v0.6.0 // indirect
 	github.com/stretchr/testify v1.6.1
 	github.com/vishvananda/netlink v1.1.1-0.20200802231818-98629f7ffc4b
 	github.com/vishvananda/netns v0.0.0-20210104183010-2eb08e3e575f // indirect
-	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad // indirect
-	golang.org/x/net v0.0.0-20210119194325-5f4716e94777
-	golang.org/x/oauth2 v0.0.0-20210210192628-66670185b0cd
-	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c
-	golang.zx2c4.com/wireguard v0.0.20201118
+	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83 // indirect
+	golang.org/x/net v0.0.0-20210220033124-5f55cee0dc0d
+	golang.org/x/oauth2 v0.0.0-20210220000619-9bb904979d93
+	golang.org/x/sys v0.0.0-20210220050731-9a76102bfb43
+	golang.org/x/text v0.3.5 // indirect
+	golang.zx2c4.com/wireguard v0.0.20200121
```

Comment:
Sir Jason needs to figure out his tagging, I couldn't. This is what "latest" brings in.